pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -121,9 +121,12 @@ public static class BrokerResourceStateModel {
 
     public static class ZkClient {
       public static final long DEFAULT_CONNECT_TIMEOUT_SEC = 60L;
+      public static final long DEFAULT_SESSION_TIMEOUT_SEC = 30L;
```

Comment:
Suggest keeping default also ms instead of seconds because it is easy to make mistake when config key is ms but value is seconds
```suggestion
      public static final long DEFAULT_CONNECT_TIMEOUT_MS = 60_000L;
      public static final long DEFAULT_SESSION_TIMEOUT_MS = 30_000L;
```